packages/cardano-services/src/StakePool/TypeormStakePoolProvider/TypeormStakePoolProvider.ts

```diff
@@ -50,9 +74,45 @@ export class TypeormStakePoolProvider extends TypeormProvider implements StakePo
     this.#lastRosEpochs = lastRosEpochs;
   }
 
+  private async getFuse(dataSource: DataSource, fuzzyOptions?: FuzzyOptions) {
+    const {
+      threshold,
+      weights: { description, homepage, name, poolId, ticker }
+    } = this.#overrideFuzzyOptions ? { ...this.#fuzzyOptions, ...fuzzyOptions } : this.#fuzzyOptions;
+
+    const cacheKey = this.#overrideFuzzyOptions
+      ? `all-metadata-${JSON.stringify([description, homepage, name, threshold, ticker])}`
+      : 'all-metadata';
+
+    return this.#cache.get(cacheKey, async () => {
```

Comment:
We need to have a strategy for updating cache so that we don't get stale values in case server is running for a really long time

Reply:
It uses the standard one created in `providerServer.ts`, I thought it was enough to behave accordingly with `DB_CACHE_TTL` value

Reply:
Got it, makes sense. Existing cache strategy will still result in some slow requests once it expires. For this use case, I think it would be better if it would always have it in cache and update it periodically (instead of when it no longer exists in cache). Not a requirement for this PR, just some thoughts for future improvement.

Reply:
LW-10063 should solve this